authors/list_latex.py

```diff
@@ -3,15 +3,16 @@
 from __future__ import print_function
 
 from json import load
+from io import open
 
 author_list = load(open("authors.json"))
 
-with open("../authors.tex", "wb") as f:
-    f.write("\\author{%\n")
+with open("../authors.tex", "w") as f:
+    f.write(u"\\author{%\n")
     for n, author in enumerate(author_list):
-        f.write(("%s%%\n" % author["name"]).encode("utf-8"))
-        f.write(("\\thanks{%s (\\email{%s}).}\n" \
-                % (author["institution"], author["email"])).encode("utf-8"))
+        f.write((u"%s%%\n" % author["name"]))
+        f.write(u"\\thanks{%s (\\email{%s}).}\n" \
+                % (author["institution"], author["email"]))
         if n < len(author_list) - 1:
-            f.write("\\and\n".encode("utf-8"))
-    f.write("}\n")
+            f.write(u"\\and\n")
+    f.write(u"}\n")
```

Comment:
Thanks for fixing it, it now looks much better, and I tested in Python 2 and it works. I couldn't get it to work before.